src/components/FormStep/index.js

```diff
@@ -818,6 +818,7 @@ const FormStep = ({
                 literals={formStep.literals}
                 canSubmitStep={canSubmit}
                 canSubmitForm={submission.submissionAllowed}
+                canSuspendForm={form.suspendAllowed}
```

Comment:
can you please update the mocks in https://github.com/open-formulieren/open-forms-sdk/blob/main/src/api-mocks/forms.js#L6